Java/PoolAndResourceFile/src/main/java/PoolAndResourceFile.java

```diff
@@ -144,7 +145,26 @@ private static CloudBlobContainer createBlobContainer(String storageAccountName,
 
         // Get a reference to a container.
         // The container name must be lower case
-        return blobClient.getContainerReference(CONTAINER_NAME);
+        return blobClient.getContainerReference(containerName);
+    }
+
+    /**
```

Comment:
descrition

Reply:
I did not get this @bgklein

Reply:
I updated the code with other review comments.